mmv1/third_party/terraform/website/docs/r/bigquery_table.html.markdown

```diff
@@ -277,12 +280,12 @@ in Terraform state, a `terraform destroy` or `terraform apply` that would delete
 
 * `mode` (Optional) - When set, what mode of hive partitioning to use when
     reading data. The following modes are supported.
-    * AUTO: automatically infer partition key name(s) and type(s).
-    * STRINGS: automatically infer partition key name(s). All types are
+  * AUTO: automatically infer partition key name(s) and type(s).
```

Comment:
This might change how the nested list is rendered. Did you check that this is the result you intended?